dashboard/management/commands/createkeys.py

```diff
@@ -67,17 +40,14 @@ def handle(self, *args, **options: dict):
         privateKey: bytes = key.exportKey()
         publicKey: bytes = key.publickey().exportKey()
 
-        jwk_obj: JWK = JWK.from_pem(publicKey)
-        public_jwk: dict = {
-            **json.loads(jwk_obj.export_public()),
-            **{'alg': 'RS256', 'use': 'sig'}
-        }
-
-        self._writeKeyFile(self.KeyFileType.PRIVATE,
-                           privateKey.decode('utf-8'))
+        result: str
 
-        self._writeKeyFile(self.KeyFileType.PUBLIC,
-                           publicKey.decode('utf-8'))
+        result = keyFileUtils.writeKeyFile(
+            keyFileUtils.KeyFileType.PRIVATE,
+            privateKey.decode('utf-8'))
+        self.stdout.write(result)
 
-        self._writeKeyFile(self.KeyFileType.JWK,
-                           json.dumps(public_jwk))
+        result = keyFileUtils.writeKeyFile(
+            keyFileUtils.KeyFileType.PUBLIC,
+            publicKey.decode('utf-8'))
+        self.stdout.write(result)
```

Comment:
Nit-pick: This is probably okay, but I don't love the reassignment to `result`. I think I'd go with separate variables.

Reply:
Interesting!  Could you explain why you prefer that?  Is it a common practice or your personal preference?

What do you think of not assigning the results to variables, and wrap the `writeKeyFile()` calls in `self.stdout.write()` directly.  That is…

```python
        self.stdout.write(keyFileUtils.writeKeyFile(
            keyFileUtils.KeyFileType.PRIVATE,
            privateKey.decode('utf-8')))
```

Reply:
Probably personal preference, but something about it seems error-prone to me. In a limited case like this, there is low risk, but in a more complicated situation, I could see a developer losing track of what the current value is. For me, whatever small gains in terms of efficiency, memory usage, or succinctness when using a single variable are outweighed by the increased clarity when using multiple variables.